src/atomate2/utils/file_client.py

```diff
@@ -375,7 +375,7 @@ def gzip(
             Overwrite gzipped file if it already exists.
         """
         path = self.abspath(path, host=host)
-        path_gz = path.with_suffix(".gz")
+        path_gz = path.parent / (f"{path.name}.gz")
```

Comment:
This looks good, but you don't need the brackets around f"{path.name}.gz"

Reply:
Ok removed